pkgs/by-name/na/nawk/package.nix

```diff
@@ -1,28 +1,19 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bison
-, buildPackages
-, installShellFiles
-}:
+{ lib, stdenv, fetchFromGitHub, bison, buildPackages, installShellFiles }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "nawk";
-  version = "20240422";
+  version = "20240728";
 
   src = fetchFromGitHub {
     owner = "onetrueawk";
     repo = "awk";
     rev = finalAttrs.version;
-    hash = "sha256-wsRkSXCLtK2jk4gW/Lpg/14NiOUANfmCrYqeKZW6CLY=";
+    hash = "sha256-LA7fdbMP3aKJ1QljoKWizqVg3ys3hd8tGaRsQnIO+Hc=";
   };
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
 
-  nativeBuildInputs = [
-    bison
-    installShellFiles
-  ];
+  nativeBuildInputs = [ bison installShellFiles ];
```

Comment:
... and this should be not needed

Reply:
* If `bison` is not in `nativeBuildInputs` the build fails with `bison command not found`
* If `installShellFiles` is not in `nativeBuildInputs` the nixf-tidy check fails due to unused import

Reply:
I was talking about the formatter, not the code.